app/src/main/res/values/strings.xml

```diff
@@ -490,6 +490,8 @@
     <string name="sync_invalid_event">Received invalid event from server</string>
     <string name="sync_invalid_task">Received invalid task from server</string>
     <string name="sync_invalid_resources_ignoring">Ignoring one or more invalid resources</string>
+    <string name="sync_notification_pending_push_title">Sync pending</string>
+    <string name="sync_notification_pending_push_message">Received a push notification from server. Waiting to sync…</string>
```

Comment:
Is the text OK or do you have suggestions? @devvv4ever @sunkup 

We can of course change it later, but we will then lose translations, so if we can keep the original string, also good.

Reply:
Maybe just:

- title: "Sync pending" (as it is)
- message: "Remote data have changed"

?

Reply:
My notification somehow got lost. I am for the latter, shorter version :+1: